dotnet/src/Connectors/Connectors.HuggingFace/HuggingFaceServiceCollectionExtensions.cs

```diff
@@ -41,6 +42,31 @@ public static class HuggingFaceServiceCollectionExtensions
             new HuggingFaceTextGenerationService(model, endpoint, apiKey, HttpClientProvider.GetHttpClient(httpClient, serviceProvider)));
     }
 
+    /// <summary>
+    /// Adds an Hugging Face chat completion service with the specified configuration.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection"/> instance to augment.</param>
+    /// <param name="model">The name of the Hugging Face model.</param>
+    /// <param name="endpoint">The endpoint URL for the chat completion service.</param>
+    /// <param name="apiKey">The API key required for accessing the Hugging Face service.</param>
+    /// <param name="serviceId">A local identifier for the given AI service.</param>
+    /// <param name="httpClient">The HttpClient to use with this service.</param>
+    /// <returns>The same instance as <paramref name="services"/>.</returns>
+    public static IServiceCollection AddHuggingFaceChatCompletion(
+        this IServiceCollection services,
+        string model,
+        Uri? endpoint = null,
+        string? apiKey = null,
+        string? serviceId = null,
+        HttpClient? httpClient = null)
+    {
+        Verify.NotNull(services);
+        Verify.NotNull(model);
+
+        return services.AddKeyedSingleton<IChatCompletionService>(serviceId, (serviceProvider, _) =>
+            new HuggingFaceChatCompletionService(model, endpoint, apiKey, HttpClientProvider.GetHttpClient(httpClient, serviceProvider)));
```

Comment:
Same here, `ILoggerFactory` should be used in this method and other methods in this file:
```suggestion
            new HuggingFaceChatCompletionService(model, endpoint, apiKey, HttpClientProvider.GetHttpClient(httpClient, serviceProvider), serviceProvider.GetService<ILoggerFactory>()));
```